lib/driver.js

```diff
@@ -333,14 +333,20 @@ class XCUITestDriver extends BaseDriver {
 
     await this.runReset();
 
+    let isLogCaptureStarted;
     const startLogCapture = async () => {
       const result = await this.startLogCapture();
       if (result) {
         this.logEvent('logCaptureStarted');
       }
       return result;
     };
-    const isLogCaptureStarted = await startLogCapture();
+
+    if (this.opts.skipLogCapture) {
+      log.info("'skipLogCapture' is set. Skipping starting logs such as crash, system, safari console and safari network.");
+    } else {
+      isLogCaptureStarted = await startLogCapture();
```

Comment:
do we start log capture separately when swkicthing to web views?

Reply:
the `startLogCapture` has starting `logs.safariConsole` and `logs.safariNetwork`. So, here is enough to control logging for webview.